src/content/docs/data-apis/manage-data/view-system-limits.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Understand limits for data ingest and querying
+title: Understand New Relic data limits
```

Comment:
@zuluecho9 I see that you're making this more generic to include sending, querying, and storing, but this new title is not easy to disambiguate from the query system limits doc. OK, I see now that in the navigation you've separated these, but the one doc is just about queries--not all limit details.